app/models/userModel.js

```diff
@@ -20,7 +20,12 @@ module.exports = (sequelize, DataTypes) => {
         field: 'last_name'
       },
       email: { type: DataTypes.STRING, allowNull: false, unique: true },
-      password: { type: DataTypes.STRING, allowNull: false }
+      password: { type: DataTypes.STRING, allowNull: false },
+      role: {
+        // eslint-disable-next-line new-cap
+        type: DataTypes.ENUM(['user', 'admin']),
```

Comment:
PostgreSQL Enum are just fine, but I recommend to use a reference table for this kind of matter in future projects. Why? Because PostgreSQL enum edition is not easy at all.